3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/CampaignAttributionEnrichment.scala

```diff
@@ -106,16 +115,18 @@ case class MarketingCampaign(
  * @param mktTerm List of marketing term parameters
  * @param mktContent List of marketing content parameters
  * @param mktCampaign List of marketing campaign parameters
+ * @param mktClick: Map of click ID parameters to networks
  */
 case class CampaignAttributionEnrichment(
   mktMedium:   List[String],
   mktSource:   List[String],
   mktTerm:     List[String],
   mktContent:  List[String],
-  mktCampaign: List[String]
+  mktCampaign: List[String],
+  mktClickId:  List[(String, String)]
   ) extends Enrichment {
```

Comment:
Could we rename these case class parameters? The names are a little confusing. Rather than `mktMedium`, I would go with `mediumParameters`, `clickIdParameters` etc.